### PR TITLE
Spark standalone mode

### PR DESCRIPTION
**Pull request by Yodafon**

### CODE REVIEW

File: GeocachingGrouping/build.gradle [hunk 1]
```diff
@@ -18,24 +18,50 @@ sourceSets {
...
 
+jar {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+    manifest {
+        attributes "Main-Class": "com.gg.core.config.AppStart"
+    }
+
+    from {
+        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
+    } {
```
Comment (vilinet) [inline GeocachingGrouping/build.gradle:29]: Ugli!

File: GeocachingGrouping/build.gradle [hunk 2]
```diff
@@ -18,24 +18,50 @@ sourceSets {
...
+//        exclude module: "spring-boot-starter-logging"
+//        exclude module: "logback-classic"
+//    }
+//    implementation('org.apache.logging.log4j:log4j-api:2.19.0')
+//    implementation('org.apache.logging.log4j:log4j-core:2.19.0')
+//    implementation('org.slf4j:slf4j-log4j12:1.7.29')
+//    implementation('jakarta.xml.bind:jakarta.xml.bind-api:2.3.2')
+//    implementation('org.glassfish.jaxb:jaxb-runtime:2.3.2')
+//}
+
```
Comment (vilinet) [inline GeocachingGrouping/build.gradle:64]: Will you ever need it?

**Review by vilinet — CHANGES_REQUESTED** [review]

File: GeocachingGrouping/build.gradle (hunk 2 above)
Comment (Yodafon) [reply GeocachingGrouping/build.gradle:64]: Spark has two different modes: Local mode and standalone mode, this dependency list is for standalone mode. In cloud or on any dedicated servers the standalone mode is the best approach.

File: GeocachingGrouping/build.gradle (hunk 1 above)
Comment (Yodafon) [reply GeocachingGrouping/build.gradle:29]: why? Suggestion?

**Review by vilinet — APPROVED** [review]

**Pull request merged**